### PR TITLE
Python 3.11 materials: Move all code to scripts

### PR DESCRIPTION
**Pull request by gahjelle**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

**Comment by gahjelle** [comment]
Lint errors will be fixed when #271 is merged

**Comment by gahjelle** [comment]
Hmm ... there are also errors from Flake8. I'm checking in with @dbader about the best way to work around these.

**Comment by gahjelle** [comment]
We're looking into moving CI checks to GitHub Actions, which will allow us to run 3.11 checks: #289

File: python-311/README.md [hunk 1]
```diff
@@ -237,30 +234,20 @@ See [Reference the Active Traceback Consistently](https://realpython.com/python3
...
-9.000000000000002
-
->>> 729**(1/3)
-8.999999999999998
-
->>> math.pow(729, 1/3)
-8.999999999999998
+```console
+(venv) $ python cube_root.py 
+math.cbrt(729) = 9.000000000000002
```
Comment (martin-martin) [inline python-311/README.md:239]: I'm still getting 9.0 for this, even after updating to the rc:

```
$ python --version
Python 3.11.0rc1
$ python cube_root.py
math.cbrt(729) = 9.0
729 ** (1 / 3) = 8.999999999999998
math.pow(729, 1 / 3) = 8.999999999999998
```

Why does it show a more precise float representation for you?

**Review by martin-martin — APPROVED** [review]
Aside from the linter errors that you mentioned and are in the process of solving, this LGTM, with the repeated note about `9.0` though

File: python-311/README.md (hunk 1 above)
Comment (gahjelle) [reply python-311/README.md:239]: That is really weird. You're on macOS, right? Could it somehow be system-dependent? I've just as consistently seen the full band of 0s ...

File: python-311/README.md [hunk 2]
```diff
@@ -126,19 +136,15 @@ $ python read_toml.py python_info.toml tomli_pyproject.toml
...
-...     "author": {"name": "Geir Arne Hjelle", "email": "geirarne@realypython.com"}}
-
->>> print(tomllib_w.dumps(data))
+```console
+(venv) $ python write_toml.py 
 url = "https://realpython.com/python311-tomllib/"
 
 [author]
 name = "Geir Arne Hjelle"
-email = "geirarne@realypython.com"
```
[the remark this answers is not in the document]
Comment (gahjelle) [reply python-311/README.md:141]: It could be fun to write for _Really Python_ as well :joy:

**Pull request merged**